projects/control-service/cicd/deploy-testing-pipelines-service.sh

```diff
@@ -51,22 +51,21 @@ if [ "$RUN_ENVIRONMENT_SETUP" = 'y' ]; then
 
   if [ -n "$DOCKERHUB_READONLY_USERNAME" ]; then
     dockerhub_secretname='secret-dockerhub-docker'
-    kubectl create secret docker-registry "$dockerhub_secretname" \
+    kubectl create secret docker-registry secret-dockerhub-docker \
                          --namespace="cicd-deployment" \
                          --docker-server="https://index.docker.io/v1/" \
-                         --docker-username="$DOCKERHUB_READONLY_USERNAME" \
+                         --docker-username="vmwaivanov" \
                          --docker-password="$DOCKERHUB_READONLY_PASSWORD" \
                          --docker-email="versatiledatakit@groups.vmware.com" --dry-run=client -o yaml | kubectl apply -f -
 
     kubectl patch serviceaccount default -p '{"imagePullSecrets":[{"name":"'$secret_name'"},{"name":"'$dockerhub_secretname'"}]}'
 
   fi
-
 fi
 
 # this is the internal hostname of the Control Service.
 # Since all tests (gitlab runners) are installed inside it's easier if we use it.
-export CONTROL_SERVICE_URL=${CONTROL_SERVICE_URL:-"http://cicd-control-service-svc:8092"}
+export CONTROL_SERVICE_URL=${CONTROL_SERVICE_URL:-"http://cicd-control-service-svc.cicd.svc.cluster.local:8092"}
```

Comment:
@tozka 
This value is set but never used as far as i can see. Do you think it can be removed?

Reply:
It's used by this line below : 

```
envsubst < $VDK_OPTIONS > $VDK_OPTIONS_SUBSTITUTED
```

This generate vdk-options from this "template" ([control-service/cicd/vdk-options.ini](https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/cicd/vdk-options.ini#L12)) which are passed to Control Service .